<ui version="4.0" >
 <class>WirelessEncryptionBase</class>
 <widget class="QWidget" name="WirelessEncryptionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Encryption</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item>
    <widget class="QComboBox" name="netSelector" />
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Encryption:</string>
       </property>
       <property name="buddy" >
        <cstring>encrypt</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="encrypt" >
       <item>
        <property name="text" >
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Open</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Shared key</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>WPA-PSK</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>WPA-EAP</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="encrypt_type" >
     <item>
      <property name="text" >
       <string>Passphrase</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Multiple keys</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pskPwLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Passphrase:</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passphrase" >
     <property name="echoMode" >
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="wpa_psk_encrypt" >
     <item>
      <property name="text" >
       <string>TKIP</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>AES</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="multkeys" >
     <property name="title" >
      <string>Multiple keys</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="bit64" >
          <property name="text" >
           <string>64 Bit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bit128" >
          <property name="text" >
           <string>128 Bit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="key1_check" >
          <property name="text" >
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="key1" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="key2_check" >
          <property name="text" >
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="key2" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="key3_check" >
          <property name="text" >
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="key3" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="key4_check" >
          <property name="text" >
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="key4" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="EAPBox" >
     <property name="title" >
      <string>WPA Enterprise</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item>
       <widget class="QComboBox" name="WPAtype" >
        <item>
         <property name="text" >
          <string>TLS</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>TTLS</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>PEAP</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="identityLabel" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Identity:</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="buddy" >
         <cstring>identity</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="identity" />
      </item>
      <item>
       <widget class="QLabel" name="anonIdentityLabel" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Anonymous Identity:</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="buddy" >
         <cstring>anon_ident</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="anon_ident" />
      </item>
      <item>
       <widget class="QLabel" name="clientCertLabel" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Client certificate:</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="buddy" >
         <cstring>client_cert</cstring>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="client_cert" />
        </item>
        <item>
         <widget class="QToolButton" name="client_cert_selector" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="clientKeyLabel" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Client Key:</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="client_key" />
        </item>
        <item>
         <widget class="QToolButton" name="client_key_selector" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="ident_passwordLabel" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Password:</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="buddy" >
         <cstring>ident_password</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ident_password" >
        <property name="echoMode" >
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="clientKeyPwLabel" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Client Key Password:</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="buddy" >
         <cstring>client_key_pw</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="client_key_pw" >
        <property name="echoMode" >
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="serverCertLabel" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Server certificate:</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="buddy" >
         <cstring>server_cert</cstring>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="server_cert" />
        </item>
        <item>
         <widget class="QToolButton" name="server_cert_selector" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="authLabel" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Authentication:</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="buddy" >
         <cstring>authentication</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="authentication" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>4</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
